lib/realtime/tenants/connect.ex

```diff
@@ -354,7 +333,20 @@ defmodule Realtime.Tenants.Connect do
         %{replication_connection_reference: replication_connection_reference} = state
       ) do
     Logger.warning("Replication connection has died")
-    {:stop, :shutdown, state}
+    Process.send_after(self(), :recover_replication_connection, 100)
+    {:noreply, state}
```

Comment:
Should we set the `replication_connection_reference` and `replication_connection_pid` to `nil` here?